docs/browser_permissions.md

```diff
@@ -0,0 +1,100 @@
+# Elevated Browser Permissions
+
+support.mozilla.org has certain elevated permissions in Firefox,
```

Comment:
I would rephrase this to focus on what these permissions do and not that sumo has elevated perms.

Reply:
I added a clause to the intro explaining what the permissions do, how's that?

Reply:
I would still rephrase it a bit, on how to interact with the browser and not about the perms that the platform has